android/src/main/java/com/google/android/cameraview/Camera1.java

```diff
@@ -1504,9 +1504,20 @@ private void stopMediaRecorder() {
         }
     }
 
-    private void setCamcorderProfile(CamcorderProfile profile, boolean recordAudio) {
+    private boolean isCompatibleWithDevice(int fps) {
+        ArrayList<int[]> validValues = (ArrayList<int[]>) mCameraParameters.getSupportedPreviewFpsRange();
```

Comment:
should we expose valid fps for the user?

Reply:
@sibelius yes. I agree and I will work on the feature tomorrow. Thanks a lot.